src/ray/raylet/worker_pool.h

```diff
@@ -472,7 +473,7 @@ class WorkerPool : public WorkerPoolInterface, public IOWorkerPoolInterface {
   /// TODO(scv119): replace dynamic options by runtime_env.
   const std::vector<std::string> &LookupWorkerDynamicOptions(StartupToken token) const;
 
-  void KillIdleWorker(std::shared_ptr<WorkerInterface> worker, int64_t last_time_used_ms);
+  void KillIdleWorker(std::shared_ptr<WorkerInterface> worker, absl::Time last_time_used);
```

Comment:
[discussion] do you think we should make this an std::optional<absl::Time>?

Reply:
Not too meaningful, for `absl::Time` we could use the default value as invalid value.
And, though unlikely, but if we fail to remove a worker, the `last_time_used` will emplaced into `idle_of_all_languages_`, so we have to figure out a way how to represent invalid last time.

Reply:
2 cents: in general I feel std::optional makes the intention more clear than default value or things like -1.

Reply:
Let me know if you want to change it, I don't have strong opinion about it.